app/src/components/altcha/altcha-widget.tsx

```diff
@@ -83,11 +79,9 @@ const AltchaWidget = forwardRef<{ value: string | null }, AltchaWidgetProps>(({
                 '--altcha-color-border-focus': '#E0E0E0',
                 '--altcha-border-radius': '4px',
             }}
-            debug
-            strings={JSON.stringify(localization)}
-            challengeurl={"/api/public/captcha/challenge/"}
+            {...(debuggingEnabled ? { debug: true } : {})}
```

Comment:
Ich vermute folgender Fix funktioniert aufgrund des HTML-Widgets nicht:
```suggestion
            debug={debuggingEnabled}
```

Reply:
Exactly, the Svelte widget only checks for the existence of the debug attribute which is even the case if I set it as undefined. The suggested change is what I tried first. :)